src/data.js

```diff
@@ -206,6 +206,33 @@ module.exports = [
       'TypeScript',
     ],
   },
+  {
+    name: 'Anthony Del Rosario',
+    description: 'Experienced Full Stack Software Engineer & Computers Lover',
+    url: 'https://adelrosarioh.me/uses',
```

Comment:
I get a 404 when I go to this page

Reply:
Hi @HugoDF!, yeah sorry about that. I was moving my site from Github Pages to Netlify. Now its online.

Thanks for reviewing.